astropy_bot/copyedit_checker.py

```diff
@@ -0,0 +1,34 @@
+from baldrick.plugins.github_pull_requests import pull_request_handler
+
+
+# TODO: we need to note somewhere that if the copyeditor changes, this needs to
+# be updated!
+COPYEDITOR_GH_HANDLE = 'lglattly'
```

Comment:
Maybe this can be a configurable item in `pyproject.toml`? If other packages decides to deploy this in their own repo, you don't want `lglattly` to get spammed.

Reply:
haha! Right.

Reply:
Although, it might be a way to get @lglattly some more work ;)